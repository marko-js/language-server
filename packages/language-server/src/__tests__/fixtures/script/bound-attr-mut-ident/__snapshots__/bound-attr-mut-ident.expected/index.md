## Hovers
### Ln 3, Col 6
```marko
  1 | <let/a = 1/>
  2 |
> 3 | <let/b := a/>
    |      ^ const b: number
  4 | //   ^?   ^?
  5 | <div onClick() {
  6 |   a++;
```

### Ln 3, Col 11
```marko
  1 | <let/a = 1/>
  2 |
> 3 | <let/b := a/>
    |           ^ const a: number
  4 | //   ^?   ^?
  5 | <div onClick() {
  6 |   a++;
```

