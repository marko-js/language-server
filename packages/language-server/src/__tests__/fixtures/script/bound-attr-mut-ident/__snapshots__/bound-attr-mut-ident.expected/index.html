<div data-marko-node-id="2" onClick></div>